modules/gdscript/editor/gdscript_highlighter.cpp

```diff
@@ -76,6 +76,10 @@ Dictionary GDScriptSyntaxHighlighter::_get_line_syntax_highlighting_impl(int p_l
 	int in_declaration_param_dicts = 0; // The number of opened `{` inside func params.
 	int in_type_params = 0; // The number of opened `[` after type name.
 
+	int square_bracket_level = 0; // for ()
+	int curly_brace_level = 0; // for {}
+	int paren_level = 0; // for []
```

Comment:
Per [comment style guide](https://contributing.godotengine.org/en/latest/engine/guidelines/code_style.html#comment-style-guide).
```suggestion
	int square_bracket_level = 0; // for ().
	int curly_brace_level = 0; // for {}.
	int paren_level = 0; // for [].
```